pyface/ui/qt4/tasks/dock_pane.py

```diff
@@ -138,8 +138,8 @@ def _get_size(self):
 
     # Trait change handlers ------------------------------------------------
 
-    @on_trait_change("dock_area")
-    def _set_dock_area(self):
+    @observe("dock_area")
+    def _set_dock_area(self, _=None):
```

Comment:
I think we have settled on the following as the recommended practice now -

```suggestion
    def _set_dock_area(self, event):
```